frontend/packages/operator-lifecycle-manager/src/components/descriptors/types.ts

```diff
@@ -23,6 +23,8 @@ export enum SpecCapability {
   arrayFieldGroup = 'urn:alm:descriptor:com.tectonic.ui:arrayFieldGroup:',
   select = 'urn:alm:descriptor:com.tectonic.ui:select:',
   advanced = 'urn:alm:descriptor:com.tectonic.ui:advanced',
+  fieldGroupThirdLevel = 'urn:alm:descriptor:com.tectonic.ui:fieldGroupThirdLevel:',
+  fieldGroupSecondLevel = 'urn:alm:descriptor:com.tectonic.ui:fieldGroupSecondLevel:',
```

Comment:
It's unclear to me why we'd need to add a descriptor here since this is all calculated from the schema. We're essentially adding an API. Would this ever be used be an operator or is it strictly internal?